app/components/spin-button.js

```diff
@@ -80,7 +80,7 @@ export default Ember.Component.extend({
     }
   },
 
-  inFlightDidChange: function() {
+  inFlightDidChange: Ember.observer(function() {
```

Comment:
Hi @ivanvanderbyl there was actually bug, updated commit yesterday.
What exactly you want me to do? Without that change it's not possible to use addon with ember cli 1.13.15. 
Before doing any refactoring it would be nice to have some tests, current setup don't work for me.

cheers,
Arek